samples/exemplar/src/main/resources/exemplar-common.yaml

```diff
@@ -1,6 +1,8 @@
 web:
   port: 8080
   idle_timeout: 30000
+  admin_dashboard:
+    protobuf_definition_url: "https://example.com"
```

Comment:
Is this a URL prefix/would you derive a specific definition URL given this one, and how?

Reply:
Yep, a prefix and we'll append the fully qualified proto name, e.g:

`https://example.com/com.squareup.exemplar.protos.HelloWebRequest`

Reply:
Suggest adding code-level docs to that effect (and maybe adding a `_prefix` to this field).

Reply:
There are some comments here, I'll clarify and provide an example: https://github.com/cashapp/misk/pull/1881/files#diff-3c3e885f03acdd9f9b5dda55fe404d5d77314b4dcc68dcb2db089c62a9f8ffb9R156-R158